debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -181,6 +182,16 @@ private void testProcessNotNullColumns(TemporalPrecisionMode temporalMode) throw
         final SourceRecord record = consumer.remove();
         VerifyRecord.isValidUpdate(record, "pk", 1);
         VerifyRecord.isValid(record);
+        final Struct before = ((Struct)record.value()).getStruct("before");
+        if (before != null) {
```

Comment:
Can it ever be null?

Reply:
@gunnarmorling protobuf and wal2josn don't behave same